operator/pom.xml

```diff
@@ -414,6 +414,11 @@
       <artifactId>simplestub</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+        <groupId>commons-cli</groupId>
+        <artifactId>commons-cli</artifactId>
+        <version>${commons-cli-version}</version>
```

Comment:
You don't need <version> here if the dependency is at top-level.

Reply:
Moved the dependency  to top level.